### PR TITLE
Rework field to match blast

### PR DESCRIPTION
**Pull request by bradfordcondon**
This PR is the sibling of  https://github.com/tripal/tripal_analysis_blast/pull/29 this PR in analysis blast

* ensure the field loads if AJAX is turned on 
* move JS out of the PHP
* put viewer in an accordian

@spficklin this PR isnt quite done but i was actually wondering if the accordian makes sense for interpro: would you ever have more than one analysis?

### CODE REVIEW

**Comment by spficklin** [comment]
> i was actually wondering if the accordian makes sense for interpro

Good question.  I can see that if someone updates the analysis you could have the older version and the newer version. I would hope that someone would remove the old and leave the updated new, but I can see a case where someone might want to keep both....  I can be convinced that it is probably not necessary.

**Comment by almasaeed2010** [comment]
Hi all,

I see in this PR that it introduces jQueryUI as a new dependency (this is different than jQuery). Any reason to use jQueryUI when collapsible fieldsets are supported by default? 

jQuery UI is an issue for us since styling it is really difficult and keeping the current style won't integrate well with our theme. 

Note: looks like this issue is also present on [tripal_analysis_blast](https://github.com/tripal/tripal_analysis_blast/pull/29).

Any thoughts?

File: tripal_analysis_interpro.install [hunk 1]
```diff
@@ -441,3 +441,25 @@ function tripal_analysis_interpro_update_7304() {
     throw new DrupalUpdateException('Could not perform update: '. $error);
   }
 }
+
+
+/**
+ * Turn on auto_attach for the Interpro field to prevent AJAX problems on entity first load.
+ */
+function tripal_analysis_blast_update_7305() {
```
Comment (bradfordcondon) [inline tripal_analysis_interpro.install]: oops, this needs to be interpro update
```suggestion
function tripal_analysis_interpro_update_7305() {
```

**Comment by bradfordcondon** [comment]
>jQuery UI is an issue for us since styling it is really difficult and keeping the current style won't integrate well with our theme.


after discussing with @spficklin we will switch to fieldsets instead.  Thanks for raising the issue @almasaeed2010

**Comment by bradfordcondon** [comment]
this PR should be ready for review.

**Comment by bradfordcondon** [comment]
@chunhuaicheng @almasaeed2010 either of you able to review?

**Comment by almasaeed2010** [comment]
Tested with and without AJAX enabled + different combos of `hide fields when empty` and it all worked as expected for me. 

Thanks @bradfordcondon

**Pull request merged**